### PR TITLE
🐛 Avoids raising error when composing disk usage

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

From real world usage (used and free are differente):

```python
{
  "HOST":           DiskUsage(used=70385278976, free=466223489024, total=536608768000, used_percent=13.12),
  "INPUTS_VOLUMES": DiskUsage(used=70385111040, free=466223656960, total=536608768000, used_percent=13.12)
}
```

It means that measurements cn be different between two parallel calls (which is feasible since the disk could be used).

A different approach to raising an error would be to remove fields which are not supported (filter them out).

Bonus: deprecated unused code

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py [hunk 1]
```diff
@@ -170,18 +166,12 @@ async def _monitor(self) -> None:
...
-                f"Currently only  the following are supported: {_SUPPORTED_ITEMS}"
-            )
-            raise RuntimeError(msg)
+        supported_usage = {k: v for k, v in usage.items() if k in _SUPPORTED_ITEMS}
 
         # notify only when usage changes
-        if self._last_usage != usage:
-            await self._publish_disk_usage(usage)
-            self._last_usage = usage
+        if self._last_usage != supported_usage:
```
Comment (pcrespov) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py:172]: Did you consider that `supported_usage` can be empty?

**Review by pcrespov — APPROVED** [review]
In this PR, instead of raising, you are filtering and ignoring. right?

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py (hunk 1 above)
Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py:172]: The tests currently cover this eventuality. A key of type `MountPathCategory.HOST` is always present.

**Comment by GitHK** [comment]
> In this PR, instead of raising, you are filtering and ignoring. right?

correct. I have also updated the description of the PR

**Review by odeimaiz — APPROVED** [review]
👍

**Review by sanderegg — APPROVED** [review]

**Pull request merged**